xds/internal/balancer/clusterresolver/clusterresolver.go

```diff
@@ -73,11 +77,10 @@ func (bb) Build(cc balancer.ClientConn, opts balancer.BuildOptions) balancer.Bal
 	}
 
 	b := &clusterResolverBalancer{
-		bOpts:    opts,
-		updateCh: buffer.NewUnbounded(),
-		closed:   grpcsync.NewEvent(),
-		done:     grpcsync.NewEvent(),
-
+		bOpts:                opts,
```

Comment:
did anything change here? Can't figure out the change in line 80-83

Reply:
There was a line break earlier, i believe it got removed. Updated the PR now.